controller.rst

```diff
@@ -181,7 +181,7 @@ If you're serving HTML, you'll want to render a template. The ``render()``
 method renders a template **and** puts that content into a ``Response``
 object for you::
 
-    // renders templates/lucky/number.html.twig
+    // render templates/lucky/number.html.twig
```

Comment:
I prefer the previous style. @javiereguiluz explained that quite well. This comment explains what the code below does, not what you as a developer should do.